package.json

```diff
@@ -26,6 +26,7 @@
     "atom-package-deps": "^5.1.0",
     "atom-text-buffer-point": "^1.0.0",
     "atom-text-buffer-range": "^1.0.0",
+    "execa": "^1.0.0",
```

Comment:
The [latest version of `execa`](https://www.npmjs.com/package/execa?activeTab=versions) is 2.0.5, but I'm using 1.0.0 to avoid extra dependencies in yarn.lock. This version of execa has already been there because it's a sub-dependency.